modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-bare-metal-config-yaml-{context}"]
+= Sample `install-config.yaml` file for bare metal
+
+You can customize the `install-config.yaml` file to specify more details about
+your {product-title} cluster's platform or modify the values of the required
+parameters.
+
+[source,yaml]
+----
+apiVersion: v1beta3
+baseDomain: example.com <1>
+compute:
+- name: worker
+  replicas: 1
+controlPlane:
+  name: master
+  replicas: 1
+metadata:
+  name: test <2>
+networking:
+  clusterNetworks:
```

Comment:
The latest installer version uses this stanza instead:

```
networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
```

It's now `clusterNetwork` and `hostPrefix`, with the latter expecting a different value than what it replaces.

Reply:
@abhinavdahiya, will this be correct for bare metal?

Reply:
let me get back to you with an example :)

Reply:
```yaml
apiVersion: v1beta4
baseDomain: example.com <1>
compute:
- name: worker
  replicas: 1
controlPlane:
  name: master
  replicas: 3 # we recommend 3 control plane hosts.
metadata:
  name: test <2>
networking:
  clusterNetworks:
  - cidr: 10.128.0.0/14 <3>
    hostPrefix: 23 <4>
 networkType: OpenShiftSDN # k8s networking type
 serviceNetwork: # IP address pool to use for service IPs. NOTE: currently only one entry is supported.
 - 172.30.0.0/16
platform:
  none: {}
pullSecret: '{"auths": ...}' <5>
sshKey: 'ssh-ed25519 AAAA...' <6>
```

annotated the fields that should be changed